sensing/pointcloud_preprocessor/schema/ring_passthrough_filter.schema.json

```diff
@@ -0,0 +1,101 @@
+{
```

Comment:
Is this file a mistake?
The parameters is for ring outlier filter, but the name is passthrough filter. 
Also you already create a passthrough_filter.schema.json so I think you can delete this file?

Reply:
Thank you for reminder! Already delect it.